custom_components/tuya_local/devices/hama_radiator_controller.yaml

```diff
@@ -0,0 +1,32 @@
+name: Hama Radiator Controller
+products:
+  - id: 176592/176593
```

Comment:
This is not the normal format for a Tuya product_id.  If you took this from iot.tuya.com, the value `product_id` under "Device Management" / "Query Device Details" is what should go here.